app/models/solr_document.rb

```diff
@@ -17,4 +17,19 @@ class SolrDocument
   # and Blacklight::Document::SemanticFields#to_semantic_values
   # Recommendation: Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
+
+  # Produces a list of authors with links to the frontend app for each.
+  def linked_authors
+    tuples = self['author_labels_tsim'].zip(self['authors_ssim'])
+    parts = tuples.map do |author|
+      "<a href=\"#{search_link(author.last)}\">#{author.first}</a>".html_safe
```

Comment:
Confirming: you need this `.html_safe` *and* the one on line 27? I'd have naïvely assumed the one on line 27 would take care of it, once all the strings in the list of tuples are joined into a single string.

Reply:
`Array#to_sentence` knows nothing about html safety, so we need the one on line 27, however, we probably don't need the one on line 25.